data_models/install.sh

```diff
@@ -16,6 +16,8 @@ psql data_refinery -c 'ALTER USER data_refinery_user CREATEDB;'
 # See https://www.howtogeek.com/177621/the-beginners-guide-to-iptables-the-linux-firewall/
 # for more information.
 iptables -A INPUT -s 172.17.0.0/16 -j ACCEPT
+/sbin/iptables-save > /etc/iptables/rules
```

Comment:
Do you need to manually write output to `/etc/iptables/rules`? The [reference](https://www.howtogeek.com/177621/the-beginners-guide-to-iptables-the-linux-firewall/) from the comment above says you just need `sudo /sbin/iptables-save` for Ubuntu.